### ISSUE TITLE
Compilation of NESTML synapse model fails when `random_normal()` function is used

### ISSUE THREAD

**Issue by pnbabu** [nest/nestml#804]
```
parameters:
    #STP
    U real = random_normal(0.5, 0.25)
    ...
end
```
When a NESTML synapse model has a `random_normal()` function, as shown above, the generated code for the synapse fails to compile with the following error.

```bash
/tmp/nestml-stdp-stp/stdp_stp_nestml__with_iaf_psc_delta_nestml.h:704:68: error: use of undeclared identifier 'get_thread'
P_.U = ((0.5) + (0.25) * normal_dev_( nest::get_vp_specific_rng( get_thread() ) )); // as real

make[2]: *** [CMakeFiles/nestml_module_module.dir/nestml_module.o] Error 1
make[1]: *** [CMakeFiles/nestml_module_module.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by pnbabu** [comment]
Fixed by #805

**Comment by pnbabu** [comment]
Closed by mistake. Reopening.
